ui/src/js/components/shared/header/branches/BranchMenu.js

```diff
@@ -184,6 +298,29 @@ class BranchMenu extends Component {
             isSticky={props.isSticky}
             branchMutations={this.state.branchMutations}
```

Comment:
no need to use this when state is has been destructured
```suggestion
            branchMutations={state.branchMutations}
```